views/listings.erb

```diff
@@ -2,7 +2,7 @@
   <% @listings.each do |listing| %>
     <li>
       <%= listing.id %> <br>
```

Comment:
perhaps don't need to display the id to the customer?

Reply:
removed the id